fava/core/charts.py

```diff
@@ -165,11 +165,16 @@ def net_worth(self, interval):
         txn = next(transactions, None)
         inventory = Inventory()
 
+        today = datetime.datetime.today().replace(
+            hour=0, minute=0, second=0, microsecond=0)
         for date in self.ledger.interval_ends(interval):
             while txn and txn.date < date:
                 for posting in filter(lambda p: p.account.startswith(types),
                                       txn.postings):
-                    inventory.add_position(posting)
+                    inventory.add_amount(
+                        posting.units,
+                        Cost(0, posting.cost.currency, today, None)
```

Comment:
The 0 should be a `Decimal`, so use the constant from `beancount.core.number`

Reply:
Done.